oracle-plugin/docs/Oracle-postaction.md

```diff
@@ -3,7 +3,7 @@
 
 Description
 -----------
-Runs a Oracle query at the end of the pipeline run.
+Runs Oracle query at the end of the pipeline run.
```

Comment:
Runs Oracle -> Runs an Oracle

Reply:
Thanks, Albert. Is there any rule of thumb that we may stick to. For example we use an/a when referencing something for the first time in text and use 'the' or no article in following sentences. 

Should we use articles in new paragraph when writing about the same thing? Also, why don't we use articles in @Description annotations?

Reply:
This is just a grammar thing where you don't say 'Runs query', but 'Runs a query'.

Reply:
This still needs to be fixed btw

Reply:
done